org.gtk.Gtk3theme.Materia.json

```diff
@@ -0,0 +1,44 @@
+{
+  "id": "org.gtk.Gtk3theme.Materia",
+  "branch": "3.22",
+  "runtime": "org.gnome.Sdk",
+  "build-extension": true,
+  "sdk": "org.gnome.Sdk",
+  "runtime-version": "3.24",
```

Comment:
Do these not work on 3.26? Or is it just that they've only been tested on 3.24.

Reply:
Oh, I didn't care much about `runtime-version` because most other Flathub's Gtk3themes used the same version (`3.24`). I will update to 3.26. Thanks.

Reply:
It should be meaningless and they could honestly use the freedesktop runtime.. which might be more logical